.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+---
+
+<!--
+Thanks for filing a 🐛 bug report 😄!
+-->
+
+<!-- Provide a general summary of the issue in the title above -->
+
+**Steps**
+<!-- The steps to reproduce the bug. -->
+1.
+2.
+3.
+
+**Problem**
+<!-- A clear and concise description of what the bug is. -->
+<!-- including what currently happens and what you expected to happen. -->
+
+**Possible Solution(s)**
+<!-- Not obligatory, but suggest a fix/reason for the bug, -->
+<!-- or ideas how to implement the addition or change -->
+
+**Notes**
+
+Output of `cargo version --verbose`:
+Output of `rustc --version`:
+Output of `rustup show active-toolchain`:
```

Comment:
Asking everyone to run 3 commands I think is too much.  `cargo -V` normally also tells you which version of rust is used (unless it is a weird non-rustup distribution).  And the majority of the time the toolchain doesn't matter.  I would like to see this simplified (perhaps just `cargo -V` until we can get an extended command).  The extra commit-hash info in the verbose flag is already in the short version.

Reply:
https://github.com/rust-lang/cargo/issues/6161

Reply:
@ehuss I opted for `cargo version` rather than `cargo -V` as I find `cargo -V` to be a bit opaque.

Reply:
I agree!